starter/js/atm.js

```diff
@@ -1,12 +1,44 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
+$( document ).ready(function() {
+    var availBalChecking = 0;
+    var availBalSavings = 0;
+    var transactionAmount = 0;
+    $('#depositChecking').click(function() {
+      transactionAmount = parseFloat($('#amountChecking').val());
+      availBalChecking += transactionAmount;
```

Comment:
Even though `availableBalanceChecking` might feel like a long variable name, it's more readable than `availBalChecking`.